include/roboteam_utils/Circle.h

```diff
@@ -0,0 +1,97 @@
+//
+// Created by emiel on 24-02-20.
+//
+
+#ifndef RTT_CIRCLE_H
+#define RTT_CIRCLE_H
+#include "Rectangle.h"
+#include "Vector2.h"
+
+namespace rtt {
+/**
+ * @brief Represents a circle by storing the center and the radius
+ * @date 24-02-2020
+ * @author Emiel Steerneman
+ */
+class Circle {
+   public:
+    Vector2 center;
+    double radius;
+
+    /**
+     * @brief Constructs a circle with a default center and radius
+     */
+    Circle();
+    /**
+     * @brief Constructs a circle from a center and a radius
+     * @param center Center of the circle
+     * @param radius Radius of the circle
+     */
+    Circle(const Vector2& center, double radius);
+    /**
+     * @brief Construct a circle from another circle by copying its center and radius
+     * @param other The circle from which to copy the center and radius
+     */
+    Circle(const Circle& other);
```

Comment:
You can probably default construct this.

Reply:
I'll look into it